packages/transport-websockets/src/index.ts

```diff
@@ -113,11 +135,13 @@ class WebSockets implements Transport<WebSocketsDialEvents> {
 
     const socket = await this._connect(ma, options)
     const maConn = socketToMaConn(socket, ma, {
-      logger: this.logger
+      logger: this.logger,
+      metrics: this.metrics?.dialerEvents
     })
     this.log('new outbound connection %s', maConn.remoteAddr)
 
     const conn = await options.upgrader.upgradeOutbound(maConn, options)
+    this.metrics?.dialerEvents.increment({ upgrade_success: true })
```

Comment:
do we need an `upgrade_error` somewhere?

Reply:
Sure, makes sense.